splunk/src/main/java/org/apache/calcite/adapter/splunk/search/SplunkConnectionImpl.java

```diff
@@ -58,7 +58,7 @@
 
   private static final Pattern SESSION_KEY =
       Pattern.compile(
-          "<response>\\s*<sessionKey>([0-9a-f]+)</sessionKey>\\s*</response>");
+          "<sessionKey>(.+)<\\/sessionKey>");
```

Comment:
Not familiar with Splunk's session yet, however as the implementation of Regex dot `.` in Java is based on longest matching, I am not sure if it's safe enough to use it here. Is `([0-9a-f]+)` not able to match the content in this case?

Reply:
The session key from Splunk contains numbers, a-z in both lower and upper cases, also there are special characters such as `^` and `_`. So `([0-9a-f]+)` could not match connections from the latest version of Splunk.  Is `([0-9a-zA-Z^_]+)` acceptable in case of safety?

Reply:
Yes, please use that one. Thanks for your explanation. :)

Reply:
No problem, I will close this pull request first.

Reply:
If you meant to change the PR, you can update your own source branch https://github.com/LastSmile732/calcite/tree/CALCITE-3067 then the PR should be able to sync the changes automatically. This way you don't have to close and file a new one.